static/css/restyle.less

```diff
@@ -1131,3 +1131,7 @@ button.good {
   height: auto;
   width: auto;
 }
+
+.manage-button {
+  margin: 1em 0 0;
```

Comment:
It's the same number of characters as `margin: 1em 0 0;`, but because I'd say this really shouldn't be a `<p>` (but I'm trying not to modify templates too much), I figured being explicit so it doesn't pick up `p` styles from elsewhere was good.